sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -171,14 +173,14 @@ def _get_epk_range_for_prefix_partition_key(self, pk_value: list) -> Range:
         if min_epk == _MaximumExclusiveEffectivePartitionKey:
             return Range("FF", "FF", True, False)
 
-        max_epk = min_epk + "FF"
+        max_epk = str(min_epk) + "FF"
         return Range(min_epk, max_epk, True, False)
 
     def _get_effective_partition_key_for_hash_partitioning(self) -> str:
-        """We shouldn't be supporting V1"""
-        pass
+        # We shouldn't be supporting V1
+        return ""
```

Comment:
I've changed the return type here from `None` to `""` - not sure whether this will have an impact on behaviour

Reply:
probably not since we don't support it anyway